HEN_HOUSE/egs++/geometry/examples/example_egs_lattice.geom

```diff
@@ -0,0 +1,113 @@
+/*
+###############################################################################
+#
+#  EGSnrc egs++ lattice geometry
+#  Copyright (C) 2019 Rowan Thomson and Martin Martinov
+#
+#  This file is part of EGSnrc.
+#
+#  egs_lattice is free software: you can redistribute it and/or modify it under
```

Comment:
Oops, the licence text here should not be modified, i.e., should say "EGSnrc is free software..." (same thing in next paragraph). We don't want to be updating that text in every file, so we chose to write: "This file is part of EGSnrc. EGSnrc is free software...".